cdap-security/src/main/java/co/cask/cdap/security/store/DummySecureStore.java

```diff
@@ -29,31 +29,37 @@
  * A dummy class that is loaded when the user has set the provider to "kms" but the cluster does not
  * have the required libraries. All operations on this class throw an UnsupportedOperationException.
  */
-public class DummyKMSStore implements SecureStore, SecureStoreManager {
+public class DummySecureStore implements SecureStore, SecureStoreManager {
 
-  private static final String UNSUPPORTED_ERROR_MSG = "Installed version of Hadoop does not support KMS. " +
-    "KMS backed secure store depends on org.apache.hadoop.crypto.key.kms.KMSClientProvider being available. " +
-    "This is supported in Apache Hadoop 2.6.0 and up and on distribution versions that are based " +
-    "on Apache Hadoop 2.6.0 and up.";
+  private static final String SECURE_STORE_SETUP = "Secure store is not configured. To use secure store the provider " +
+    "needs to be set using the \"security.store.provider\" property in cdap-site.xml. " +
+    "The value could either be \"kms\" for Hadoop KMS based provider or \"file\" for Java JCEKS based provider. " +
+    "Both without quotes. " +
+    "KMS based provider is supported in distributed mode for Apache Hadoop 2.6.0 and up and on distribution versions " +
+    "that are based on Apache Hadoop 2.6.0 and up. " +
+    "Java JCEKS based provider is supported in In-Memory and Standalone modes. If the provider is set to file then " +
+    "the user must provide the password to be used to access the keystore. The password can be set using " +
+    "\"security.store.file.password\" property in cdap-security.xml. " +
+    "For more details please refer to https://wiki.cask.co/display/CE/Using+Secure+Store.";
```

Comment:
this link should perhaps move to docs.cask.co. actually do we need these links? Can we just remove them? The information you have above is already quite verbose.